backend/schemas/source/base/Base.libsonnet

```diff
@@ -1,465 +1,453 @@
-      },
-    },   
-  ],
-};
-
-local Atoms = {
-
-};
-
-local Enum = {
-  YorN: Types.string {
-    // description: 'A yes or no answer',
-    enum: [
-      Const.Y,
-      Const.N,
-    ],
-    //title: 'YorN'
-  },
-  NA: Types.string {
-    //description: 'A 'not applicable' answer',
-    enum: [
-      Const.NA,
-    ],
-    title: Const.NA,
-  },
-  EmptyString_Zero_Null: {
-    description: 'Empty string, zero, or null expected.',
-    enum: [
-      Const.empty_string,
-      0,
-      Const.NULL,
-    ],
-    title: 'EmptyString_Zero_Null',
-  },
-  EmptyString_Null: {
-    description: 'Empty string or null',
-    enum: [
-      Const.empty_string,
-      Const.NULL,
-    ],
-    title: 'EmptyString_Null',
-  },
-  EmptyString_EmptyArray_Null: {
-    description: 'Empty string, empty array, or null',
-    enum: [
-      Const.empty_string,
-      Const.empty_array,
-      Const.NULL,
-    ],
-    title: 'EmptyString_Null',
-  },
-  UnitedStatesStateAbbr: Types.string {
-     'enum': [
-        'AL',
-        'AK',
-        'AZ',
-        'AR',
-        'CA',
-        'CO',
-        'CT',
-        'DE',
-        'FL',
-        'GA',
-        'HI',
-        'ID',
-        'IL',
-        'IN',
-        'IA',
-        'KS',
-        'KY',
-        'LA',
-        'ME',
-        'MD',
-        'MA',
-        'MI',
-        'MN',
-        'MS',
-        'MO',
-        'MT',
-        'NE',
-        'NV',
-        'NH',
-        'NJ',
-        'NM',
-        'NY',
-        'NC',
-        'ND',
-        'OH',
-        'OK',
-        'OR',
-        'PA',
-        'RI',
-        'SC',
-        'SD',
-        'TN',
-        'TX',
-        'UT',
-        'VT',
-        'VA',
-        'WA',
-        'WV',
-        'WI',
-        'WY'
-     ],
-  },
-  AuditPeriod: Types.string {
-     description: 'Period type of audit being submitted',
-     enum: [
-        'annual',
-        'biennial',
-        'other'
-     ],
-     title: 'AuditPeriod',
-  },
-  AuditType: Types.string {
-      description: 'Type of audit being submitted',
-      enum: [
-          'program-specific',
-          'single-audit',
-      ],
-      title: 'AuditType',
-  },
-  MajorProgramAuditReportType: Types.string {
-    description: 'Major program report types',
-    enum: [
-      'U',
-      'Q',
-      'A',
-      'D',
-    ],
-    title: 'MajorProgramAuditReportType',
-  },
-  OrganizationType: Types.string {
-    description: 'Org type',
-    enum: [
-      'state',
-      'local',
-      'tribal',
-      'higher-ed',
-      'non-profit',
-      'unknown',
-      'none',
-    ],
-    title: 'OrganizationType',
-  },
-  SubmissionStatus: Types.string {
-    description: 'Submission status',
-    enum: [
-      'in_progress',
-      'submitted',
-      'received',
-      'available',
-    ],
-    title: 'SubmissionStatus',
-  },
-  ALNPrefixes: Types.string {
-    description: 'Valid two-digit program numbers; part of the CFDA/ALN',
-    enum: [
-      '1',
```

Comment:
Did you mean to remove these? I thought they were going to be two-digit with leading zeroes to match formulas like [this one](https://github.com/GSA-TTS/FAC/blob/c04acb04167ce86b44514e6f686739d283bace63/backend/schemas/output/excel/json/federal-awards-expended-template.json#L35). Or am I missing a change elsewhere covering that?